docs/src/manual/running.md

```diff
@@ -1,6 +1,45 @@
 # Running IJulia
 
 
+## Command-Line Launcher
+
+Starting with Julia 1.12, you can launch IJulia directly from the command line using the `ijulia` command. First, you need to install the app entry point by running in the Julia REPL:
+
+```julia
+pkg> app add IJulia
```

Comment:
I think IJulia is one of the few packages that still has a build step 😅 Shall we just install the app automatically when building? My only concern is that from reading the docs it seems that an app is somehow hardcoded to a specific Julia binary? That's a bit unfortunate because we recently added a juliaup integration that reduced the need for rebuilding IJulia to once per minor release (#1201). And perhaps if it's that experimental we should leave it out by default, can always enable it later.

Reply:
That sounds ok, as long as it still installs the same kernels as now. @stevengj, what do you think?